etna/core/models/mixins.py

```diff
@@ -133,4 +138,38 @@ class Meta:
         )
     ]
 
+
+class RequiredHeroImageMixin(models.Model):
+    """Mixin to add hero_image attribute to a Page, and make it required."""
+
+    hero_image = models.ForeignKey(
+        get_image_model_string(),
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+
+    hero_image_caption = RichTextField(
+        verbose_name="hero image caption (optional)",
+        features=["bold", "italic", "link"],
+        blank=True,
+        help_text=(
+            "An optional caption for hero images. This could be used for image sources or for other useful metadata."
+        ),
+    )
+
+    class Meta:
+        abstract = True
+
+    content_panels = [
+        MultiFieldPanel(
+            [
+                FieldPanel("hero_image"),
+                FieldPanel("hero_image_caption"),
+            ],
+            heading="Hero image",
+        )
+    ]
+

```

Comment:
Inherited from HeroImageMixin, so...

```suggestion
    content_panels = [
        MultiFieldPanel(
            [
                FieldPanel("hero_image"),
                FieldPanel("hero_image_caption"),
            ],
            heading="Hero image",
        )
    ]
```

Reply:
Hi Andy, I'm not sure if you've tried these on your local - but when I tried to do this it didn't work for some reason - I kept getting a load of errors. I'll give them another go now though but I did think it was odd that I couldn't just inherit it this way..

Reply:
```python
SystemCheckError: System check identified some issues:

ERRORS:
articles.ArticlePage: (models.E005) The field 'id' from parent model 'core.requiredheroimagemixin' clashes with the field 'id' from parent model 'wagtailcore.page'.
articles.ArticlePage: (wagtailcore.E002) Manager does not inherit from PageManager
        HINT: Ensure that custom Page managers inherit from wagtail.models.PageManager
collections.TimePeriodExplorerIndexPage: (models.E005) The field 'id' from parent model 'core.requiredheroimagemixin' clashes with the field 'id' from parent 
model 'wagtailcore.page'.
collections.TimePeriodExplorerIndexPage: (wagtailcore.E002) Manager does not inherit from PageManager
        HINT: Ensure that custom Page managers inherit from wagtail.models.PageManager
collections.TimePeriodExplorerPage: (models.E005) The field 'id' from parent model 'core.requiredheroimagemixin' clashes with the field 'id' from parent model 'wagtailcore.page'.
collections.TimePeriodExplorerPage: (wagtailcore.E002) Manager does not inherit from PageManager
        HINT: Ensure that custom Page managers inherit from wagtail.models.PageManager
collections.TopicExplorerIndexPage: (models.E005) The field 'id' from parent model 'core.requiredheroimagemixin' clashes with the field 'id' from parent model 'wagtailcore.page'.
collections.TopicExplorerIndexPage: (wagtailcore.E002) Manager does not inherit from PageManager
        HINT: Ensure that custom Page managers inherit from wagtail.models.PageManager
collections.TopicExplorerPage: (models.E005) The field 'id' from parent model 'core.requiredheroimagemixin' clashes with the field 'id' from parent model 'wagtailcore.page'.
collections.TopicExplorerPage: (wagtailcore.E002) Manager does not inherit from PageManager
        HINT: Ensure that custom Page managers inherit from wagtail.models.PageManager